test/org/apache/coyote/http11/TestHttp11InputBuffer.java

```diff
@@ -687,24 +691,6 @@ public void testInvalidEndOfRequestLine01() {
     }
 
 
-    @Test
```

Comment:
I would leave this test in, but change its nature (i.e. require that the response code is 200). Or is your assertion that the "correct" test is below in `test/org/apache/coyote/http11/TestHttp11InputBufferCRLF.java `?

Reply:
Yes, the correct test is below.